assets/queries/terraform/kubernetes_pod/service_account_name_undefined_or_empty/test/positive.tf

```diff
@@ -0,0 +1,115 @@
+resource "kubernetes_pod" "test1" {
```

Comment:
Can you split these samples into separate tf files?
```
positive1.tf
positive2.tf
etc...
```
Then adjust the `positive_expected_result.json` for example:

```json
	{
		"queryName": "Service Account Name Undefined Or Empty",
		"severity": "MEDIUM",
		"line": 6, <--- replace the new line in the new file
		"fileName": "positive1.tf"
	},
```